crates/uv/tests/run.rs

```diff
@@ -233,6 +233,7 @@ fn run_script() -> Result<()> {
     Reading inline script metadata from: main.py
     Resolved 1 package in [TIME]
     Prepared 1 package in [TIME]
+    Installing to environment at [CACHE_DIR]/environments-v1/[RANDOM]/[RANDOM]/bin/python
```

Comment:
I'm hesitant to display these as user-facing messages for these internal environments — I wonder if there's a good way to avoid that?

Reply:
Ah so revert the `[RANDOM]` filter and have an additional conditional to ensure we don't print in the event it's an internal environment e.g; `[CACHE]/environments-v1...`?